src/Native/Runtime/CachedInterfaceDispatch.cpp

```diff
@@ -554,22 +554,4 @@ COOP_PINVOKE_HELPER(void*, RhpGetTailCallTLSDispatchCell, ())
     return (void *)(&RhpTailCallTLSDispatchCell);
```

Comment:
`RhpTailCallTLSDispatchCell` should be only used by castable. I think it can be deleted too.

Reply:
Does `RhpTailCallTLSDispatchCell` and `RhpGetTailCallTLSDispatchCell` is the same? Looking on the assembly, I cannot remove `RhpSetTLSDispatchCell` since it is used in other places.

Reply:
> I cannot remove `RhpSetTLSDispatchCell` since it is used in other places.

Which place is problematic? From a quick search, this is called from places related to CastableObject (that can be deleted).

Reply:
It set `t_TLS_DispatchCell` which is used in assembly inside `GET_TLS_DISPATCH_CELL/SET_TLS_DISPATCH_CELL`  macro. Upon closer examination, seems to be I miss that this macro is used only in `RhpSetTLSDispatchCell/RhpGetTailCallTLSDispatchCell` so I can remove `RhpSetTLSDispatchCell`. Shoud I ?